stock_picking_backorder_reason/models/stock_backorder_reason.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Camptocamp SA
+# Copyright 2018 Jacques-Etienne Baudoux (BCIM sprl) <je@bcim.be>
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockBackorderReason(models.Model):
+
+    _name = "stock.backorder.reason"
+    _description = "Stock Backorder Reason"
+
+    name = fields.Char(required=True, translate=True)
+    backorder_action_to_do = fields.Selection(
+        selection=[
+            ("create", "Create backorder"),
+            ("cancel", "Cancel backorder"),
+            ("use_partner_option", "Use partner option"),
+        ],
+        default="picking_type",
```

Comment:
@rousseldenis `picking_type` is not into the selection list

Reply:
Done